tests/multipleBackend/routes/routeBackbeat.js

```diff
@@ -223,7 +230,7 @@ function getMetadataToPut(putDataResponse) {
     return mdToPut;
 }
 
-describe('backbeat routes', () => {
+describeSkipIfAWS('backbeat routes', () => {
```

Comment:
I don't understand why we skip here, and esp. based on some AWS condition.
Backbeat routes are present on both Zenko and S3C, so should be tested... and the test here seem to apply to both...